### DISCUSSION TITLE
How to reload schema cache?

### DISCUSSION THREAD

**Discussion by mathiasrscom** [supabase/supabase#2935] [Questions] [answered]
Hi, 

I'm getting the following error message, when trying to run the dev server using `yarn dev`: 

`{
  hint: 'If a new foreign key between these entities was created in the database, try reloading the schema cache.',
  message: 'Could not find a relationship between products and prices in the schema cache'
}`

I'm using the Stripe Subscription template provided by Supabase in the SQL section and the repo: 
https://github.com/vercel/nextjs-subscription-payments

How and where can I reload the schema cache?

**Comment by steve-chavez** [comment, marked as answer]
Hey @mathiassdev,

Can you try doing:

```sql
NOTIFY pgrst, 'reload schema';
```

The above should reload PostgREST schema cache.

**Reply by jesper-bylund** [reply]
How is this used? Where can one trigger this?

**Reply by steve-chavez** [reply]
On any SQL client

**Reply by GaryAustin1** [reply]
@jesper-bylund  and specifically the SQL editor in the dashboard if you are not using your own 3rd party client.

**Reply by JUDE-EQ** [reply]
cool, still working today

**Reply by usman-javed1** [reply]
how i can do this from nodejs

**Reply by GaryAustin1** [reply]
Write a postgres function and call it with rpc.
